cl/recap/tasks.py

```diff
@@ -2265,21 +2289,22 @@ def fetch_pacer_case_id_and_title(s, fq, court_id):
     return {}
 
 
-def fetch_docket_by_pacer_case_id(session, court_id, pacer_case_id, fq):
-    """Download the docket from PACER and merge it into CL
-
-    :param session: A PacerSession object to work with
-    :param court_id: The CL ID of the court
-    :param pacer_case_id: The pacer_case_id of the docket, if known
-    :param fq: The PacerFetchQueue object
+def create_or_update_docket_data_from_fetch(
+    fq: PacerFetchQueue,
+    court_id: str,
+    pacer_case_id: str,
```

Comment:
```suggestion
    pacer_case_id: str | None,
```
Considering that `pacer_case_id` can be `None` for appellate purchases.

Reply:
You're right!